check/pytest

```diff
@@ -19,23 +19,23 @@ cd "$(git rev-parse --show-toplevel)"
 
 PYTEST_ARGS=()
 ACTUALLY_QUIET=""
+CIRQ_ONLY=""
```

Comment:
Can we document this command line flag in the doc string at the top?

Reply:
Yes, I can add that - this flag already exists though, so I'll add it in a separate PR to keep this smaller.